src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -931,4 +931,8 @@ nuget client-certs List</value>
   <data name="SourcesCommandProtocolVersionDescription" xml:space="preserve">
     <value>The NuGet server protocol version to be used. Currently supported versions are 2 and 3. See https://learn.microsoft.com/nuget/api/overview for information about the version 3 protocol. Defaults to 2 if not specified.</value>
   </data>
+  <data name="SourcesCommandAllowInsecureConnectionsDescription" xml:space="preserve">
+    <value>Enables HTTP connections, applicable only to the `Add` and `Update` subcommands. Note: Data sent or received is not secure. Please refer to https://aka.ms/nuget-https-everywhere.</value>
```

Comment:
Some users have reported links not working if the message ends in a period.  We might want to omit it in this case or add more words at the end like "for more information."